Tech-Journal-ConsoleApp/Tech-Journal.Test/TestingEmailEntry.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using NUnit.Framework;
+using Tech_Journal_ConsoleApp;
+using System.Net.Mail;
+
+namespace Tech_Journal.Test
+{
+    [TestFixture]
+    public class TestingEmailEntry
```

Comment:
I think the naming convention of 'NameOfClass**Tests**' would be easier to read.